nixos/modules/config/users-groups.nix

```diff
@@ -703,7 +703,7 @@ in {
     '';
 
     # Warn about user accounts with deprecated password hashing schemes
-    system.activationScripts.hashes = {
+    system.activationScripts.hashes = if !config.boot.initrd.systemd.enable then {
```

Comment:
Seems like this warning should make it to somewhere better whether systemd initrd is enabled or not, rather than simply hiding it when systemd initrd is enabled.

Reply:
Not sure if we still need this script at all tbh. Do you have anything in mind where this should go?

Reply:
Not really :/ There isn't really a place that every user is guaranteed to see an import warning like this.